bindings/stronghold-nodejs/README.md

```diff
@@ -13,6 +13,49 @@ Development Release: this version matches the dev branch of this repository, may
 ```bash
 npm install @iota/identity-stronghold-nodejs@dev
 ```
+## Usage
+<!-- 
+Test this example using https://github.com/anko/txm: `txm README.md`
+
+Replace imports with local paths for txm:
+!test program
+cat \
+| sed -e "s#require('@iota/identity-wasm/node')#require('../wasm/node/identity_wasm.js')#" \
+| sed -e "s#require('@iota/identity-stronghold-nodejs')#require('./dist/index.js')#" \
+| node
+-->
+<!-- !test check Nodejs Example -->
+```javascript
+const { AccountBuilder, ExplorerUrl } = require('@iota/identity-wasm/node')
+const { Stronghold } = require('@iota/identity-stronghold-nodejs')
+
+
+async function main() {
+    
+    const stronghold = await Stronghold.build("./example-strong.hodl", "my-password", true);
+    
+    // The creation step generates a keypair, builds an identity
+    // and publishes it to the IOTA mainnet.
+    let builder = new AccountBuilder({
+        storage: stronghold,
+    });
+    let account = await builder.createIdentity();
+
+    // Retrieve the DID of the newly created identity.
+    let did = account.did();
+
+    // Print the DID of the created Identity.
+    console.log(did.toString())
+
+    // Print the local state of the DID Document
+    console.log(account.document());
+
+    // Print the Explorer URL for the DID.
+    console.log(`Explorer Url:`, ExplorerUrl.mainnet().resolverUrl(did));
```

Comment:
This should ideally use `JSON.stringify` to avoid printing opaque `[Object]` entries. Not sure if that matters for a short example, maybe it's distracting?

```suggestion
    // Print the DID of the newly created identity.
    const did = account.did();
    console.log(did.toString());

    // Print the local state of the DID Document.
    const document = account.document();
    console.log(JSON.stringify(document, null, 2));

    // Print the Explorer URL for the DID.
    console.log(`Explorer URL:`, ExplorerUrl.mainnet().resolverUrl(did));
```